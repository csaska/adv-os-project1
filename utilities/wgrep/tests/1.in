version https://git-lfs.github.com/spec/v1
oid sha256:22112022335631f153d275d77b282404e99c29df9bc8850efc61323ccc0ea8ac
size 76
