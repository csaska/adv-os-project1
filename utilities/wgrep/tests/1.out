version https://git-lfs.github.com/spec/v1
oid sha256:a9a40956907c3a5b45c425800f47b158c7b9424e5b2fd85b23843a22db150408
size 33
